views/score.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Get a word score!</h1>
+<form id="get-scores" action="#" method="post">
+  <input type="text" id="word" name="word">
+  <input type="submit" name="submit" id="submit" value="Score it">
+</form>
+<p>
```

Comment:
Might be good to wrap this paragraph section in a Ruby if statement like `<% if @word %>`